src/core/ui.js

```diff
@@ -59,14 +59,29 @@ respecPill.addEventListener("click", function(e) {
   this.setAttribute("aria-expanded", String(menu.hidden));
   menu.hidden = !menu.hidden;
 });
+document.documentElement.addEventListener("keyup", e => {
```

Comment:
```suggestion
document.addEventListener("keyup", e => {
```

I wonder if this should be more specific tho... maybe `body`?

Reply:
AFAIK `window.addEventListener("keyup")`, `document.addEventListener("keyup")`, `document.body.addEventListener("keyup")` etc. works all the same way.

Reply:
I was thinking more from a logical/debugging perspective... make the listener easier to find, reduced potential conflicts with other things higher in the tree.

Reply:
In other words, we should attach this directly on the target... maybe even remove the listener when the menu is hidden... re-attach when shown.

Reply:
>make the listener easier to find

I should admit this is reasonable because `document.addEventListener` doesn't appear in the DOM tree view.

Reply:
>we should attach this directly on the target

Isn't the purpose here opening ReSpec menu when user presses `i` wherever the user is?

Reply:
Well, that's something @pradeepgangwar is proposing... not something I'm 100% we should go with yet. What's more important is the up and down keys and tabbing from one option to the next.